lint/src/test/java/permissions/dispatcher/Utils.java

```diff
@@ -2,15 +2,16 @@
 
 import org.intellij.lang.annotations.Language;
 
-final class Utils {
+public final class Utils {
```

Comment:
I used to have different packages for `psi` and `uast` implementations, which made the visibility change of this class necessary. Now that everything ended up in 1 package after all, I will probably revert this back.